### PR TITLE
[site2] Fix swagger of SourceBase and SinkBase display problem

### PR DESCRIPTION
**Pull request by wolfstudy**
Signed-off-by: xiaolong.ran <ranxiaolong716@gmail.com>

### Motivation

There are some display problems and there is some ambiguity in the description information output, as follows:

#### Source

-  [Creates a new Pulsar Source in cluster mode](https://pulsar.apache.org/source-rest-api/#operation/registerSource) 
- [Updates a Pulsar Source currently running in cluster mode](https://pulsar.apache.org/source-rest-api/#operation/updateSource)

#### Sink

- [Creates a new Pulsar Sink in cluster mode](https://pulsar.apache.org/sink-rest-api/#operation/registerSink)
- [Updates a Pulsar Sink currently running in cluster mode](https://pulsar.apache.org/sink-rest-api/#operation/updateSink)

In the pull request, will fix them, the display after fix is as follows:

image

### CODE REVIEW

**Comment by wolfstudy** [comment]
@Jennifer88huang @sijie @jerrypeng This pull request is similar to the change #4901 , PTAL thanks.

File: pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/impl/SinksBase.java [hunk 1]
```diff
@@ -258,15 +257,15 @@ public void deregisterSink(@ApiParam(value = "The sink's tenant")
     )
     @ApiResponses(value = {
             @ApiResponse(code = 400, message = "Invalid request"),
-            @ApiResponse(code = 404, message = "The sink does not exist"),
+            @ApiResponse(code = 404, message = "The Pulsar Sinj does not exist"),
```
Comment (Jennifer88huang) [inline pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/impl/SinksBase.java]: Sinj --> Sink

**Comment by Jennifer88huang** [comment]
Does "Sink" and "Source" here refer exclusively to "Pulsar Sink" and "Pulsar Source"? Or they also refer to other (external) systems?

**Comment by wolfstudy** [comment]
> Does "Sink" and "Source" here refer exclusively to "Pulsar Sink" and "Pulsar Source"? Or they also refer to other (external) systems?

In here, Sink and Source are similar to Function and are the general term for all external systems.

**Comment by wolfstudy** [comment]
ping @sijie @Jennifer88huang @jerrypeng PTAL and approve this, thanks.

**Comment by sijie** [comment]
run integration tests

**Comment by wolfstudy** [comment]
run integration tests

```
11:19:06.130 [kafka-producer-network-thread | b2a68b7b-3cf4-45ad-b69b-0682fe7127c2:org.apache.kafka.clients.NetworkClient@748] WARN  org.apache.kafka.clients.NetworkClient - [Producer clientId=b2a68b7b-3cf4-45ad-b69b-0682fe7127c2] Connection to node 1 (localhost/127.0.0.1:35686) could not be established. Broker may not be available.
11:19:06.240 [ForkJoinPool.commonPool-worker-11:org.apache.pulsar.tests.integration.utils.DockerUtils@143] ERROR org.apache.pulsar.tests.integration.utils.DockerUtils - Error reading dir from container TestFileSystemOffload-wmear-pulsar-bookie-1
java.io.IOException: gzip finished without exhausting source
	at org.testcontainers.shaded.okio.GzipSource.read(GzipSource.java:100) ~[testcontainers-1.11.2.jar:?]
	at org.testcontainers.shaded.okio.RealBufferedSource$1.read(RealBufferedSource.java:439) ~[testcontainers-1.11.2.jar:?]
	at org.apache.pulsar.tests.integration.utils.DockerUtils.dumpContainerDirToTargetCompressed(DockerUtils.java:136) ~[test-classes/:?]
	at org.apache.pulsar.tests.integration.containers.PulsarContainer.beforeStop(PulsarContainer.java:101) ~[test-classes/:?]
	at org.apache.pulsar.tests.integration.containers.ChaosContainer.stop(ChaosContainer.java:64) ~[test-classes/:?]
	at java.util.stream.ForEachOps$ForEachOp$OfRef.accept(ForEachOps.java:184) [?:1.8.0_191]
	at java.util.ArrayList$ArrayListSpliterator.forEachRemaining(ArrayList.java:1382) [?:1.8.0_191]
	at java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:481) [?:1.8.0_191]
	at java.util.stream.ForEachOps$ForEachTask.compute(ForEachOps.java:291) [?:1.8.0_191]
	at java.util.concurrent.CountedCompleter.exec(CountedCompleter.java:731) [?:1.8.0_191]
	at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:289) [?:1.8.0_191]
	at java.util.concurrent.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1056) [?:1.8.0_191]
	at java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1692) [?:1.8.0_191]
	at java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:157) [?:1.8.0_191]
```

**Pull request merged**